src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,70 @@
+.contacts {
+  margin-bottom: 64px;
```

Comment:
The BEM unit must not have any external margins

Reply:
Done